tool/tsh/app.go

```diff
@@ -245,15 +249,25 @@ func formatAppConfig(tc *client.TeleportClient, profile *client.ProfileStatus, a
 	} else {
 		uri = fmt.Sprintf("https://%v:%v", appPublicAddr, port)
 	}
-	curlCmd := fmt.Sprintf(`curl \
-  --cacert %v \
+
+	var curlCmd string
+	if tc.InsecureSkipVerify {
```

Comment:
Can we simplify this to something like:

```go
	var curlInsecureFlag string
	if tc.InsecureSkipVerify {
		curlInsecureFlag = "--insecure "
	}

	curlCmd := fmt.Sprintf(`curl %s\
  --cert %v \
  --key %v \
  %v`,
			curlInsecureFlag,
			profile.AppCertPath(appName),
			profile.KeyPath(),
			uri)

```

I think it's easier to read and to understand that the only difference is the `--insecure` flag